### DISCUSSION TITLE
Interrogate memory map using scripting

### DISCUSSION THREAD

**Discussion by aytey** [x64dbg/x64dbg#2972] [Q&A]
I'm interested in setting a scripted breakpoint at an address that comes from reading the memory map:

![image](https://user-images.githubusercontent.com/14277540/201501115-919b1094-9402-499c-b671-3f57d3d7c24d.png)

For example, I want to be able to do:

```
bph 00007FF67EFA1000
```

where `00007FF67EFA1000` is the value that comes from the page information labelled ` ".text"`.

Is this something that's accessible via scripting?

**Comment by jjdroid** [comment]
Sweet. That sounds super easy! Have fun!

**Comment by mrexodia** [comment]
You can use `mod.main()` to get the base of the main module. Then +0x1000 is usually the text section.
